doc/en/user/source/community/backuprestore/installation.rst

```diff
@@ -8,7 +8,7 @@ Manual Install
 
 To download and install the required extensions by hand:
 
-#. Download the geoserver-2.15-SNAPSHOT-backup-restore-plugin.zip from:
+#. Download the geoserver-2.19-SNAPSHOT-backup-restore-plugin.zip from:
```

Comment:
I would make this more generic so that we do not mention a version explicitly hence this sentence does not need be update continuosly.

In the end there are 3 versions live at any moment, stable, maintenance and master so we could word this better.

Reply:
Yeah that's definitely a good idea. I'll look into this